misc/userscripts/iframe

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import os
+import sys
+import bs4
+import re
+
+
+class StaticGetAttrType(type):
+    """Metaclass which simply allows class to define
+    a __getstaticattr__ method.
+    __getstaticattr__ is the static equivalent of __getattr__.
+    """
+    def __getattr__(cls, name):
+        return cls.__getstaticattr__(name)
+
+
+class Env(metaclass=StaticGetAttrType):
+    """Helper class which allows to access environment variables"""
+    def __getstaticattr__(name):
+        return os.environ[name]
+
+
+def read(path):
+    """Reads every line of a file
+
+    Return:
+        content or None
+    """
+    with open(path) as f:
+        return '\n'.join(f.readlines())
+
+
+def qexec(*cmds):
```

Comment:
You might want to consider [python-qutescript](https://github.com/hiway/python-qutescript) instead, which also allows for nicer access to the environment variables :wink: